npc/quests/first_class/tu_sword.txt

```diff
@@ -2068,7 +1428,8 @@ geffen,154,143,3	script	Dequ'ee	734,{
 					mes "be the message I'm supposed to receive from Shurack. I need his exact message or I can't send";
 					mes "a response...";
 					close;
-				case 2:
+				}
+				else {
```

Comment:
For me `else / else if` should only be used if the code continue after the previous condition.